src/components/MapNode.tsx

```diff
@@ -97,7 +97,7 @@ export const MapNode = observer((props) => {
               {phrasings[currentPhrasingIndex].text}
             </h3>
           ) : (
-            <h4 css={s.nodeTitle(detailViewOpen)}>{phrasings[currentPhrasingIndex].text}</h4>
+            <h3 css={s.nodeTitle(detailViewOpen)}>{phrasings[currentPhrasingIndex].text}</h3>
```

Comment:
does this node title code still need to be separate from the question title code? even if the style is still different, we could use the same elements. I see the presence/absence of onClick but that seems like it should be consistent anyway?

Reply:
I think it depends on whether the top level questions turn out to be substantially different in our app. I could see it going either way. In the debate map application everything is just a node, but we may have different concerns.